docs/zh_cn/advanced_guides/datasets.md

```diff
@@ -1 +1,132 @@
-# Datasets
+# 数据集
+
+- [数据集](#数据集)
+  - [数据集](#数据集-1)
+    - [重构个人数据集](#重构个人数据集)
+    - [在个人Config中调用OpenMMLab其他代码库的数据集](#在个人config中调用openmmlab其他代码库的数据集)
+  - [采样器](#采样器)
+  - [数据变换](#数据变换)
+
+`mmselfsup` 算法库中的 `datasets` 文件夹包罗了各种与数据加载相关的模块文件。
+此文件夹主要分为如下三个部分：
+
+- 自定义数据集，用于图像读取与加载
+- 自定义数据集采样器，用于图像加载之前进行索引的读取
+- 数据变换工具，用于在数据输入模型之前进行数据增强，如 `RandomResizedCrop`
+
+在本教程中，我们将对三个部分依次进行较为详尽的解释。
+
+## 数据集
+
+OpenMMLab 开源算法体系为用户提供了海量开箱即用的数据集，这些数据集都与 [BaseDataset](https://github.com/open-mmlab/mmengine/blob/429bb27972bee1a9f3095a4d5f6ac5c0b88ccf54/mmengine/dataset/base_dataset.py#L116)一脉相承，
+并在 [MMEngine](https://github.com/open-mmlab/mmengine) 中得以实现。 如想进一步了解 `BaseDataset` 中的各项功能，感兴趣的用户可以参考 [MMEngine](https://mmengine.readthedocs.io/en/latest/advanced_tutorials/basedataset.html) 中的文档。对于`MMSelfSup`， `ImageNet`、`ADE20KDataset` 和`CocoDataset` 是三个较为常用的数据集。 在起步之前，用户需要对文件夹进行一些前置的重构工作，具体指南如下所述。
+
+### 重构个人数据集
+
+万事俱备，只欠东风。使用准备好的这些数据集，用户需要将数据集重构为如下格式。
+
+```
+mmselfsup
+├── mmselfsup
+├── tools
+├── configs
+├── docs
+├── data
+│   ├── imagenet
+│   │   ├── meta
+│   │   ├── train
+│   │   ├── val
+│   │
+│   │── ade
+│   │   ├── ADEChallengeData2016
+│   │   │   ├── annotations
+│   │   │   │   ├── training
+│   │   │   │   ├── validation
+│   │   │   ├── images
+│   │   │   │   ├── training
+│   │   │   │   ├── validation
+│   │
+│   │── coco
+│   │   ├── annotations
+│   │   ├── train2017
+│   │   ├── val2017
+│   │   ├── test2017
+```
+
+
+更为详尽的注释文件以及各子文件夹的结构，可以参考 OpenMMLab 的其他代码库，如 [MMClassfication](https://github.com/open-mmlab/mmclassification),
+[MMSegmentation](https://github.com/open-mmlab/mmsegmentation) 和 [MMDetection](https://github.com/open-mmlab/mmdetection).
+
+
+### 在个人Config中调用OpenMMLab其他代码库的数据集
```

Comment:
Should leave space between Chinese and English

Reply:
done